src/lib/types/model.ts

```diff
@@ -365,3 +365,24 @@ export interface IFeatureStrategySegment {
     featureStrategyId: string;
     segmentId: number;
 }
+
+export interface ISuggestChangeSet {
+    id: number;
+    environment: string;
+    state?: string;
+    project?: string;
+    createdBy?: string;
+    createdAt?: Date;
+    changes?: ISuggestChange[];
```

Comment:
I thought (at least) some of these options were required: definitely `project`, but probably also `changes` and the `created*` options. Why are they made optional here?

Reply:
resolved, leftover from developing. thanks!